dace/memlet.py

```diff
@@ -176,15 +177,16 @@ def to_json(self):
     @staticmethod
     def from_json(json_obj, context=None):
         ret = Memlet()
-        dace.serialize.set_properties_from_json(ret,
-                                                json_obj,
-                                                context=context,
-                                                ignore_properties={'src_subset', 'dst_subset', 'num_accesses', 'is_data_src'})
-        
+        dace.serialize.set_properties_from_json(
```

Comment:
Unrelated to the PR, but why is YAPF constantly switching the formatting of such lines?